templates/daemon.upstart.erb

```diff
@@ -0,0 +1,30 @@
+# Prometheus <%= @name %> Exporter (Upstart unit)
+description "Prometheus exporter <%= @name %>"
```

Comment:
Maybe change this to `"Prometheus <%= @name %> exporter"` for consistency with the above comment.